source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -18,25 +18,26 @@ The global vulnerability database is created automatically, currently pulling da
 This database can be configured to be updated periodically, ensuring that the solution will check for the very latest CVEs.
 
 Once the global vulnerability database (with the CVEs) is created, the detection process looks for vulnerable packages in the inventory databases (unique per agent). Alerts are generated when a CVE (Common Vulnerabilities and Exposures) affects a package that is known to be installed in one of the monitored servers. A package is labeled as vulnerable when its version is contained within the affected range of a CVE.
+The results are presented as alerts and also stored in a database. So you can check the last scan alerts, or query every single agent's Vulnerable Software DB.
+
+.. _vuln_det_scan_types:
 
 Scan types
 ==========
 
-The results are presented as alerts and also stored in a database. You can check the last scan alerts, or query each agent's individual CVEs DB.
-
-The Vulnerability Detector module can run a scan on startup (**run_on_start** option) and with a fixed frequency (**interval**).
-In any of these cases, the packages that have been already scanned will wait until the **ignore_time** expires to be re-scanned.
-This results in a better performance and we avoid a repeated alert during a configurable time. We have then two different type of scans
+The Vulnerability Detector module can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain arbitrary period of time (:ref:`interval <vuln_det_interval>`).
```

Comment:
I think _arbitrary_ makes the reader think it's a random time. Maybe we should use i.e. _configurable_.

Reply:
Or, to avoid the duplication in the line below, we can just remove _arbitrary_ or replace it with _desired_ or _adjustable_

Reply:
The arbitrary word has been removed.